great_expectations/__init__.py

```diff
@@ -1,3 +1,17 @@
+import sys
+
+PYTHON_SUPPORTED_VERSION = {"major": 3, "minor": {"lower": 8, "upper": 10}}
```

Comment:
we could use the `version` package here to make this easier.

`version.Version` has built in comparators

Reply:
I would prefer not to take on a dependency. `sys` is builtin and the conditional is not complicated.
I do see comparators builtin if you pass in a string representation to `version.Version` but I don't know how to get this representation of the python runtime without using `sys` and then doing some manipulation of `sys.version` or `sys.version_info` which seems more complicated than this.